torch/utils/data/datapipes/utils/common.py

```diff
@@ -63,3 +63,8 @@ def validate_pathname_binary_tuple(data):
         raise TypeError("pathname binary tuple should have string type pathname, but got {}".format(type(data[0])))
     if not isinstance(data[1], BufferedIOBase):
         raise TypeError("pathname binary tuple should have BufferedIOBase based binary type, but got {}".format(type(data[1])))
+
+# Warns user that the DataPipe has been moved to TorchData and will be removed from `torch`
+def deprecation_warning_torchdata(name):
+    warnings.warn(f"""{name} is deprecated and will be removed from the package `torch`. Please import it """
+                   """from the new package TorchData: https://github.com/pytorch/data""", DeprecationWarning)
```

Comment:
nit: Maybe also talk about functional API for these DataPipe also exists in TorchData.